src/app/hearing-part/containers/hearings-search/hearings-search.component.ts

```diff
@@ -155,21 +155,12 @@ export class HearingsSearchComponent implements OnInit {
         }
     }
 
-    private openDialog(actionTitle: string, notes: NoteViewmodel[]) {
```

Comment:
You have removed usage of `NoteViewmodel`, but it's still in imports. Please remove it from imports as it brake the build.

Reply:
Yep you're right! done :)